cmd/mpi-operator/main.go

```diff
@@ -70,6 +71,10 @@ func main() {
 		kubeflowInformerFactory = informers.NewSharedInformerFactoryWithOptions(kubeflowClient, 0, informers.WithNamespace(namespace), nil)
 	}
 
+	var pdbInformer policyinformers.PodDisruptionBudgetInformer
+	if !enableGangScheduling {
```

Comment:
The condition should be the other way around (same in other places)? e.g. if gang scheduling is enabled, then we initialize pdbInformer.